tools/scripts/build_base_image.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Since the base image has been pushed to docker hub, do we need this script in this repo?

Reply:
It seems that this script serves no one but the mmdeploy team.

Reply:
Yes, the script is for maintainers.